app/screens/AppNavigator/screens/Transactions/TransactionsScreen.tsx

```diff
@@ -86,7 +97,7 @@ function TransactionRow (navigation: NavigationProp<TransactionsParamList>): (ro
         }}
       >
         <View style={tailwind('w-8 justify-center items-center')}>
-          <Ionicons name='arrow-down' size={24} color={color} />
+          <MaterialCommunityIcons name='arrow-up' size={24} color={color} />
```

Comment:
Replace MaterialIcons. Should it be arrow-up? from the original it's arrow-down

Reply:
Updated to use the icon defined in view model, seems that the icon naming logic is already there but it wasn't used in this screen originally.

Also added the `React.ComponentProps<typeof MaterialIcons>['name']` as the type to further enforce valid material icon name